.github/workflows/release.yml

```diff
@@ -53,21 +53,23 @@ jobs:
 
   build_wheels_linux:
     name: Build ${{ matrix.baseimage }} wheels
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         include:
-          - baseimage: manylinux2014_x86_64
+          - os: ubuntu-latest
+            baseimage: manylinux2014_x86_64
             cibw_build: "*manylinux_x86_64"
             arch: x86_64
-          - baseimage: musllinux_1_2_x86_64
+          - os: ubuntu-latest
+            baseimage: musllinux_1_2_x86_64
             cibw_build: "*musllinux_x86_64"
             arch: x86_64
-          # Wait until native arm64 runners are available
-          # - baseimage: manylinux2014_aarch64
-          #   cibw_build: "*manylinux_aarch64"
-          #   arch: aarch64
+          - os: ubuntu-24.04-arm
+            baseimage: manylinux2014_aarch64
+            cibw_build: "*manylinux_aarch64"
+            arch: aarch64
           # - baseimage: musllinux_1_2_aarch64
           #   cibw_build: "*musllinux_aarch64"
           #   arch: aarch64
```

Comment:
It's possible to enable this too, since these numpy bdist wheels are available.

Reply:
I had just done that and it seems to work!